src/selectors/groups.js

```diff
@@ -1,8 +1,7 @@
 // Copyright 2021 Stanford University see LICENSE for license
+import _ from "lodash"
 
-export const hasGroups = (state) => {
-  Object.keys(state.entities.groupMap).length > 0
-}
+export const hasGroups = (state) => !_.isEmpty(state.entities.groupMap)
```

Comment:
This should fix problem Peter reported with tests failing locally, but working in CI.